src/components/member/tabs/contract-tab/FromDateComponent.tsx

```diff
@@ -0,0 +1,71 @@
+import { Agreement, Contract } from 'api/generated/graphql'
+import { format } from 'date-fns'
+import { Button, ButtonsGroup } from 'hedvig-ui/button'
+import { DateTimePicker } from 'hedvig-ui/date-time-picker'
+import * as React from 'react'
+import { Card } from '../../../../../shared/hedvig-ui/card'
+import {
+  changeFromDateOptions,
+  useChangeFromDate,
+} from '../../../../graphql/use-change-from-date-for-agreement'
+import { ButtonSpacing } from './contract-item'
+
+const initialFromDate = (agreement: Agreement): Date =>
+  agreement.fromDate ? new Date(agreement.fromDate) : new Date()
+
+const FromDateComponent: React.FunctionComponent<{
+  agreement: Agreement
+  contract: Contract
+}> = ({ agreement, contract }) => {
```

Comment:
Should probably pass the showNotifation function as well for user feedback
```suggestion
const FromDateComponent: React.FunctionComponent<{
  agreement: Agreement
  contract: Contract
  showNotifaction: (data: NotifcationDate) => void
}> = ({ agreement, contract }) => {
```